cloudslang-entities/src/test/java/io/cloudslang/lang/entities/DeserializeTest.java

```diff
@@ -93,4 +106,58 @@ public void testDeserializeAsyncLoopStatement() throws IOException {
         testToAndFromJson(asyncLoopStatement, AsyncLoopStatement.class);
     }
 
-}
\ No newline at end of file
+    private class GsonStringInputDeserializer extends TypeAdapter<Input> {
```

Comment:
need to unignore tests here -- do you think this is the correct approach (define adapters also for Argument, Output, Result)?